app/controllers/users_controller.rb

```diff
@@ -0,0 +1,31 @@
+class UsersController < ApplicationController
+  include UsersHelper
```

Comment:
You should fix this @patriciachrysy, also, it's in the instructions (image below) to add it as a private method as mentioned by the previous reviewer. Rails helpers are often for the views, not the controller. If you are in doubt, consider checking [this article](https://www.rubyguides.com/2020/01/rails-helpers/).

<img width="820" alt="Screenshot 2020-05-05 at 19 36 37" src="https://user-images.githubusercontent.com/31304943/81102640-d3d95480-8f07-11ea-9dfa-d62fe265840d.png">